modules/op-creating-pipeline-tasks.adoc

```diff
@@ -55,3 +55,8 @@ git-clone                                1 day ago
 s2i-python                               1 day ago
 tkn                                      1 day ago
 ----
+
+[role="_additional-resources-creating-pipeline-tasks"]
+.Additional resources
+
+* xref:../../cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc#managing-nonversioned-and-versioned-cluster-tasks[Managing non-versioned and versioned cluster tasks]
```

Comment:
Unfortunately, we can't use xrefs in modules. I recommend moving lines 58-62 to `openshift_pipelines/creating-applications-with-cicd-pipelines.adoc` and pasting it just below the `include::modules/op-creating-pipeline-tasks.adoc[...]` statement.